AK/StringFloatingPointConversions.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, Dan Klishch <danilklishch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#ifdef KERNEL
+#    error This file should not be included in the KERNEL as it deals with doubles \
+           and there is no guraantee does not do any floating point computations.
```

Comment:
I've used FloatingPointStringConversions.h as a base for this file (and it has the same omitted "it").
Maybe just completely delete pragma? For instance, AK/FloatingPoint.h does not say anything about kernel.

Reply:
I think this is more likely to (unintentionally) be used by the KERNEL when printing stuff, so I think keeping the pragma is good, perhaps only on the cpp file, so there are no implementations of it for KERNEL might be good enough

Anyway for now let's keep it, since this is still a WIP PR